app/const.py

```diff
@@ -42,7 +42,7 @@ class MIMETYPE(StrEnum):
     ),
     "carlisle.gov.uk": (
         ("Cumberland Council",),
-        ("Carlisle City Centre",),
+        ("Workington", "Cleator Moor", "Millom", "Carlisle", "Carlisle City Centre", "Maryport Town Centre"),
```

Comment:
Could there be a comment explaining that cumberland and carlisle have merged LA but both domains are still in use